docs/docs/7-migration-guide/migration-from-v7-to-v8.md

```diff
@@ -1269,6 +1269,42 @@ The new usage simplifies the component structure - children can now be passed di
 
 :::note Handled by following upgrade script
 
+```diff
+<Tooltip
+- trigger="hover"
+></Tooltip>
+```
+
+### Import `Button` from `@comet/admin` package
+
+```diff
+- import { Button } from "@mui/material";
++ import { Button } from "@comet/admin";
+```
```

Comment:
It seems to me that this is the only real change in the migration guide, could the others be left overs from merging/rebasing?